conf4j-core/src/main/java/com/sabre/oss/conf4j/internal/factory/jdkproxy/AbstractJdkProxyConfigurationInvocationHandler.java

```diff
@@ -79,13 +83,40 @@ protected AbstractJdkProxyConfigurationInvocationHandler(ConfigurationModel conf
                 .filter(p -> p instanceof SubConfigurationListPropertyModel)
                 .map(PropertyModel::getPropertyName)
                 .collect(toSet());
+
+        generateConstructor();
+    }
+
+    private void generateConstructor() {
+
+        try {
+            constructor = MethodHandles.Lookup.class.getDeclaredConstructor(Class.class, int.class);
+        } catch (NoSuchMethodException e) {
+            throw new RuntimeException(e);
+        }
+        constructor.setAccessible(true);
+
     }
 
+    private void readObject(ObjectInputStream inputStream) throws IOException, ClassNotFoundException {
```

Comment:
Once _constructor_ is not a transient filed anymore, this method is not necessary.